projects/js-packages/publicize-components/src/components/notice/index.js

```diff
@@ -0,0 +1,16 @@
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+
+const Notice = ( { children, type = 'default' } ) => {
+	const className = classnames( styles.notice, styles[ `notice--${ type }` ] );
+
+	return <div className={ className }>{ children }</div>;
+};
+
+Notice.propTypes = {
+	children: PropTypes.node.isRequired,
+	type: PropTypes.oneOf( [ 'default', 'warning' ] ),
```

Comment:
I think this should include _error_ as well, isn't it?

Reply:
Yes thanks, I forgot to update that when I added the new type.